modules/cluster-image-registry-operator.adoc

```diff
@@ -33,7 +33,16 @@ If insufficient information is available to define a complete `image-registry` r
 
 The Cluster Image Registry Operator runs in the `openshift-image-registry` namespace and it also manages the registry instance in that location. All configuration and workload resources for the registry reside in that namespace.
 
+:cluster-caps:
```

Comment:
```suggestion

```
As discussed, this will be removed. The rest of the changes LGTM.

Reply:
Yes thanks :woman_facepalming: thanks for catching!